tests/dummy/public/test-forms/basic-fields/form.ttl

```diff
@@ -158,6 +158,20 @@ ext:numericalInputField a form:Field ;
 
 ext:mainFg form:hasField ext:numericalInputField .
 
+##########################################################
+# Disabled input
+##########################################################
+ext:disabledInputField a form:Field ;
+    sh:name "Disabled input" ;
+    sh:order 223 ;
+    sh:path ext:inputValue ;
+    form:displayType displayTypes:defaultInput ;
+    form:displayShow true ;
```

Comment:
Was the predicate name of this discussed with Felix? Just double checking since it's pretty final after we merge this :smile:.

Reply:
No, see this is a draft "POC" PR ;) 
As I indeed miss the needed context and history to make a correct decision on this.